man/measure.Rd

```diff
@@ -38,10 +37,15 @@ measurev(fun.list, sep, pattern, cols, multiple.keyword="value.name",
     are considered measure variables.}
   \item{pattern}{Perl-compatible regex with capture groups to match to
     \code{cols}. Columns that match the regex are considered measure variables.}
-  \item{cols}{A character vector that is automatically provided, 
-    with all of the column names of the data table input to melt. 
-    The user should not provide this argument; 
-    to specify the columns to be measured, use the pattern argument.}
+  \item{cols}{
+    A character vector of column names.
+    When used in the context of
+    \code{melt(measure.vars=measure())}, the
+    user should not provide the \code{cols} argument, which is
+    automatically set to all of the column names of the input data
+    table, using non-standard evaluation.
+    To specify the columns to be measured,
+    use either \code{sep} or \code{pattern} argument.}
```

Comment:
here is my attempt to clarify NSE in the measure() docs, is this OK? similar wording is used in patterns().